### DISCUSSION TITLE
Supabase Storage Adapter

### DISCUSSION THREAD

**Discussion by jhb-dev** [payloadcms/payload#4936] [Feature Requests & Ideas]
I've been excited to see the integration of a [PostgreSQL adapter](https://payloadcms.com/blog/payload-2-0) in Payload 2.0, it opens up great possibilities. 

I am currently using Supabase as a backend for my PayloadCMS instance. However, I noticed that there's currently no dedicated storage adapter for Supabase. Given the growing popularity of Supabase, having a native storage adapter for it would immensely benefit developers who rely on Payload.

I wanted to draw your attention to a previous effort in this direction - a Pull Request in the old storage repository ([PR #39](https://github.com/payloadcms/plugin-cloud-storage/pull/39)). 

I believe this addition could significantly enhance Payload's versatility and appeal. Looking forward to possibly seeing this integration in future releases!

**Comment by denolfe** [comment]
This is one that we definitely should get to. Labeling as planned.
